contexts/DonationContext/tests/Unit/UseCases/AddDonation/AddDonationUseCaseTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Tests\Unit\DonationContext\UseCases\AddDonation;
+
+use ReflectionMethod;
+use PHPUnit\Framework\TestCase;
+use WMDE\Fundraising\Frontend\DonationContext\UseCases\AddDonation\AddDonationUseCase;
+
+/**
+ * @covers \WMDE\Fundraising\Frontend\DonationContext\UseCases\AddDonation\AddDonationUseCase
+ */
+class AddDonationUseCaseTest extends TestCase {
+
+	public function testGetInitialDonationStatus(): void {
+		$useCase = $this->createMock( AddDonationUseCase::class );
+		$method = new ReflectionMethod( $useCase, 'getInitialDonationStatus' );
+		$method->setAccessible( true );
```

Comment:
Reflection like this undermines encapsulation and breaks automated refactoring. Do you think this is somehow still better than what we've done so far for testing UC code?